src/applications/appeals/996/content/InformalConferenceContact.jsx

```diff
@@ -12,11 +12,10 @@ export const informalConferenceContactInfo = (
         accredited representative.
       </p>
       <p className="vads-u-margin-bottom--0">
-        <va-link
-          external
-          href="/get-help-from-accredited-representative/"
-          text="Find, contact, and appoint an accredited representative"
-        />
+        <a href="/get-help-from-accredited-representative/" target="_blank">
```

Comment:
What is the reason for replacing va-link?

Reply:
We were using `va-link` to show an `external` icon, but after a [discussion in the #platform-design-system channel](https://dsva.slack.com/archives/C01DBGX4P45/p1726839417096199) where we were told not to do that yet (until the guidance is in place). So the only way to get links to open in a new tab is to use `<a target="_blank">`